doc/main/tbb_userguide/attach_flow_graph_to_arena.rst

```diff
@@ -0,0 +1,68 @@
+.. _attach_flow_graph_to_arena:
+
+Attach Flow Graph to an Arbitrary Task Arena
+======================
+
+
+|short_name| ``task_arena`` interface provides mechanisms to guide tasks execution within
+the arena by setting the preferred computation units or restricting part of computation units.
+In some case, you may want to use mechanisms within a flow graph.
+
+During its construction a ``graph`` object attaches to the arena, in which the constructing
+thread occupies a slot. Whenever a task is spawned on behalf of the graph, it is spawned
+in the arena of the graph it is attached to, disregarding the arena of the thread
+which is caused a task to be spawned.
```

Comment:
I think that the second sentence in this paragraph better be moved closer to the second example as the example now shows this explicitly and so may raise questions of readers.

Perhaps, the best place for it is right after the example introduction words, i.e. line #41.

Reply:
Agree here

Reply:
Updated with suggested changes.